.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: php
+php:
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+#  - 7.0
+#  - hhvm
+
+env: COVERAGE=0
+
+matrix:
+  include:
+    - php: 5.6
+      env: COVERAGE=1
+
+before_script:
+  - if [[ "$COVERAGE" -eq 0 ]]; then composer install; else composer require satooshi/php-coveralls --dev; fi
+
+script:
+  - if [[ "$COVERAGE" -eq 0 ]]; then vendor/bin/phpunit; else mkdir -p build/logs; vendor/bin/phpunit --coverage-clover=build/logs/clover.xml; fi
+
+after_script:
```

Comment:
Would there be a benefit to using `after_success` instead of `after_script`, and making fewer calls to coveralls?